subsys/shell/Kconfig

```diff
@@ -318,6 +318,57 @@ config SHELL_CUSTOM_HEADER
 	  extension of the shell APIs at the macro level. Please use cautiously!
 	  The internal shell API may change in future releases.
 
+config SHELL_BYPASS_RX_BUFF_SIZE
+	int "Size of bypass receive buffer"
+	default 16
+	help
+	  Maximum size of each individual read in bypass mode.
+
+config SHELL_BYPASS_USER
+	bool "Bypass from user thread"
+	help
+	  Allow user threads to have access to bypass mode. When enabled, this
+	  feature gives a user thread exclusive access to the shell's underlying
+	  transport through a special API. While in bypass mode, the shell's
+	  functionality is temporarily disabled. When the user thread wants to
+	  return to the shell, it disables bypass mode through another special
+	  API method. In support of this feature, two pipes are created whose
+	  buffer sizes are controlled by configuration variables
+	  SHELL_BYPASS_USER_RX_PIPE_ALIGN, SHELL_BYPASS_USER_RX_PIPE_SIZE,
+	  SHELL_BYPASS_USER_TX_PIPE_ALIGN, and SHELL_BYPASS_USER_TX_PIPE_SIZE.
+
+config SHELL_BYPASS_USER_RX_PIPE_ALIGN
```

Comment:
put `if SHELL_BYPASS_USER` here instead and remove the `depends on SHELL_BYPASS_USER` for more concise file